lib/engine/game/base.rb

```diff
@@ -1495,6 +1507,11 @@ def can_par?(corporation, parrer)
         !corporation.ipoed
       end
 
+      # Base definition to not allow splitting in most games
+      def can_split?(_corporation, _parrer)
```

Comment:
why is this needed? if we switch to convert can we get of all of the split code?

Reply:
We can't get rid of all of the 1871 split code, but yes I should be able to reuse the other action and not have split code in the base.

Reply:
I think you could alternatively remove this and use `if @game.respond_to? :can_split? && @game.can_split?(...)` when you use this method?

Reply:
Sure, I will edit to that tonight.